stable/airflow/UPGRADE.md

```diff
@@ -1,5 +1,12 @@
 # Upgrading Steps
 
+## `v7.2.0` → `v7.3.0`
```

Comment:
All other upgrade headings have an `X`, for example:
`v7.2.X` → `v7.3.0`

Reply:
fixed